cli/dts/lib.deno.ns.d.ts

```diff
@@ -86,93 +112,225 @@ declare interface PerformanceMeasureOptions {
   end?: string | number;
 }
 
+/** The global namespace where Deno specific, non-standard APIs are located. */
 declare namespace Deno {
   /** A set of error constructors that are raised by Deno APIs.
+   *
+   * The can be used to provide more specific handling of failures within code
+   * which is using Deno APIs. For example, handling attempting to open a file:
+   *
+   * ```ts
+   * try {
+   *   const file = await Deno.open("./some/file.txt");
+   * } catch (error) {
+   *   if (error instanceof Deno.errors.NotFound) {
+   *     console.error("the file was not found");
+   *   } else {
+   *     // otherwise re-throw
+   *     throw error;
+   *   }
+   * }
+   * ```
    *
    * @category Errors
    */
   export namespace errors {
-    /** @category Errors */
+    /**
+     * Raised when the underlying operating system indicates that the file
+     * was not found.
+     *
+     * @category Errors */
     export class NotFound extends Error {}
-    /** @category Errors */
+    /**
+     * Raised when the underlying operating system indicates the current user
```

Comment:
It's not always coming from the OS - if you don't have `--allow-*` flag it will also raise this error